### PR TITLE
uploading/Second_Batch/Rajendar/myInfo.md

### PR DESCRIPTION
**Pull request by raju10101997**

### CODE REVIEW

File: Second_Batch/Rajendar/myInfo.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 i am Great Rajendar Uppulanche
 ******************************
+
+branch line : uploading into branch rajendar_work
```
Comment (coolyasu) [inline Second_Batch/Rajendar/myInfo.md:4]: please add address

File: Second_Batch/Rajendar/myInfo.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
 i am Great Rajendar Uppulanche
 ******************************
+
```
Comment (raju10101997) [inline Second_Batch/Rajendar/myInfo.md:3]: I am a computer engineer.

File: Second_Batch/Rajendar/myInfo.md [hunk 3]
```diff
@@ -1,2 +1,8 @@
...
+branch line : uploading into branch rajendar_work
+
```
Comment (coolyasu) [inline Second_Batch/Rajendar/myInfo.md:5]: please add edu details

**Review by coolyasu — APPROVED** [review]

File: Second_Batch/Rajendar/myInfo.md [hunk 4]
```diff
@@ -1,2 +1,8 @@
...
+
+
```
Comment (raju10101997) [inline Second_Batch/Rajendar/myInfo.md:7]: blah blah blah

File: Second_Batch/Rajendar/myInfo.md (hunk 1 above)
Comment (raju10101997) [reply Second_Batch/Rajendar/myInfo.md:4]: ok bro

**Pull request merged**